lib/screen/feralfile_home/list_exhibition_view.dart

```diff
@@ -170,7 +170,7 @@ class ListExhibitionView extends StatefulWidget {
 }
 
 class _ListExhibitionViewState extends State<ListExhibitionView> {
-  static const _padding = 12.0;
+  static const _padding = 15.0;
```

Comment:
I think we should create a const padding for all screens

Reply:
Yes, i think we can do it late.